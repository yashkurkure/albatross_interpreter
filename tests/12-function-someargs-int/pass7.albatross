fun fun7223 int (a0 int,a1 int) { return a0&a1;}
fun fun47782 int (a0 int) { return a0;}
fun fun21981 int (a0 int,a1 int) { return a0&a1;}
fun fun64995 int (a0 int,a1 int,a2 int) { return a0*a1*a2;}
fun fun42465 int (a0 int,a1 int,a2 int,a3 int) { return a0+a1+a2+a3;}
fun fun14904 int (a0 int,a1 int) { return a0+a1;}
fun fun34613 int (a0 int,a1 int,a2 int,a3 int) { return a0&a1&a2&a3;}
fun fun37030 int (a0 int,a1 int,a2 int,a3 int) { return a0&a1&a2&a3;}
fun fun8227 int (a0 int,a1 int,a2 int,a3 int) { return a0&a1&a2&a3;}
fun fun8108 int (a0 int,a1 int,a2 int,a3 int) { return a0*a1*a2*a3;}
fun fun30999 int (a0 int,a1 int) { return a0+a1;}
fun fun46812 int (a0 int,a1 int,a2 int,a3 int) { return a0+a1+a2+a3;}
fun fun16523 int (a0 int,a1 int,a2 int,a3 int) { return a0+a1+a2+a3;}
fun fun6869 int (a0 int,a1 int) { return a0|a1;}
fun fun52506 int (a0 int,a1 int) { return a0*a1;}
fun fun53161 int (a0 int,a1 int) { return a0&a1;}
fun fun9722 int (a0 int) { return a0;}
fun fun45402 int (a0 int,a1 int) { return a0&a1;}
fun fun28353 int (a0 int,a1 int,a2 int,a3 int) { return a0&a1&a2&a3;}
fun fun61818 int (a0 int,a1 int,a2 int,a3 int) { return a0|a1|a2|a3;}
fun fun31960 int (a0 int) { return a0;}
fun fun58290 int (a0 int) { return a0;}
fun fun52924 int (a0 int,a1 int,a2 int) { return a0*a1*a2;}
fun fun22740 int (a0 int) { return a0;}
fun fun6076 int (a0 int,a1 int,a2 int) { return a0*a1*a2;}
fun fun40754 int (a0 int,a1 int,a2 int) { return a0-a1-a2;}
fun fun41325 int (a0 int,a1 int,a2 int) { return a0&a1&a2;}
fun fun11949 int (a0 int) { return a0;}
fun fun53637 int (a0 int) { return a0;}
fun fun20606 int (a0 int,a1 int,a2 int,a3 int) { return a0|a1|a2|a3;}
printint(fun40754(157|119+71|146|211*184|26|156-224-219*185|87|171&244+56|63&162+138|164-141&239+184&54&59&105-27-76|230&113&192&164|44,9&124|60*182|47|5&149&46*185-202|96|11|223|55-214*60+78|99|229|120&34*179-74&120&163|108|236*59*178|115+68+51,167&161+223|69|229-93+192&113&52+169|203&114&96-93|240-233|169+18&247|158&243-75|7+248|222|203&171-200-85|236-81|207));
printint(fun61818(96|32*129|108|97*82*167+203|182|196+49*123&195-48|132*190-75&174*24|107|108|145|181-134&201|96-141+99*161&179*219|21,157*206&172|234|25*141|157&208|74&21|219+185|99*93|29+78*221-32|234+2+224|195+70*94*86&143&198-25|16&219|73+128,239*67&3-82-48|144&0|15+1&153|73-187|24|39*154|186&227|95|250+28-70-13|248|195*230*253-203|157+55-92|132&60,33&14*165*220|137*139|47+16|6-52&30-171-19|21|96|9-87&55*37-176*144&108+245+101+86|229-49|90|16*116*109|49));
printint(fun31960(13&174&175+218*209*115&207-143&216+91-213+191|22+52|55|182-147|112-22-155&236|58&148*19|89*89|0|53+98+53&75|102));
printint(fun28353(188|135-199|253+84|69-13-135-93-74+122-101*101|129|208+170&115|63*91|12*28&87*65|194|196|183&222+6|178+121|205|204,248*16-213&34|158*76&14|155|13|137&159|107&195|111-149&106&118+227*139*152|197-79|57-113-163|223-48-119-98&217|177|132,23&126+118-50|171-175+199+190+68|84+25|114|77-7|134+23&77|124|104&38+109*162-203*145+151+247&16+122|174&145&64-102,243-103+88-82*70-185+195*14|145|173&199+119&194-4+43-63+175&125|222+73+29*175|68-198&77-51|163-238|67*117-170+146));
printint(fun20606(71-166+59-144+105&44&91-141-252&113*199|109+241|233-56|187|44&202|232*149|72-63|54*241|213|119-8+7|198&150+197|74,146|158*214-217|202&41|10|208|170&208+198+163-46|27+37|8|228+189|94|208+75|202&222*238|160*133|127&7|251*53|235*69,88+3+90&155+44|59+204-120*143*162-214|188|90|250|88|131|50*69|253&27&126|246&192+209+215|214+211-66|134-17|211*139,180|112+209|142&80&8|206|44-213|125&49+180|182+150|248|96|185|59+152|205|198-121|215*53|34+136-7&151|151&246*8|172));
printint(fun22740(115-228|185-31|170*148|50|88&33-42|30|86|228|214|5|242&86-213&109|132|90*27*193|117+178+139-254+44&227&136|60&205));
printint(fun6076(194+36|163-194*9|232-187-160&220&9|8&252|102|180+150*33+29&234&10-183&199|177|28*137|145&239*228-83+193-255*159-116,150-121&197-115|87|145*80+128-165+121&140&124|217*45+81*183-140*4|53-192|23-169|131-151+99-51*174&62&193+73&158|21,178*185|132+150+241&173*220&85&74-229*241|168&113&209|248-205|217|247+130|159|71-179-91|122*145+193+12|166|148-4|4|52));
printint(fun11949(78*231*29&152|214-140+78|76&236&51*47&36|144|105-50-102&192|33|227-46*236&130|66|9&212|132+17+140&234|216*147+194));
printint(fun6869(58*80&183-69|156-132|245*30+135-99-224*118-231|57-27|42-200+22-195+173-159*88|219|184|195+85&147+30+100|131+233-1,139&104&248|156*161-231&159*51*119*52|90+162*193+215|182|86|105&110|23+57+51|4|139-5+200|178*78*251|216|54-10*126));
printint(fun6076(155|207|28|105+177-117-159-58+119-249*108+113*6|14|127|44+59+59+124&172*119&80-106-80|217&109&201-218&231+77*61|28,41&130*34*36&100+45-125|231|112|80|206-158|63+14&41-117-235|20+108-123-218-39+70|80*220|16&183|6|252&204-9|242,94*144-102+173-188|9+178+245&146*211+239+245*255|1&89+34|129-48|132&125+151&216&20*47&162&172|148+95|74|182+100-251));
exit(0);
