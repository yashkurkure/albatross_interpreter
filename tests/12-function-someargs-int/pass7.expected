-2303-17935511-1-1-131999-1360959999
